_posts/intro/2014-02-01-raspberry-pi-quick-start.md

```diff
@@ -38,58 +36,47 @@ in order to enable it to follow the instructions available at the following URL:
 
 {% include alerts.html message="The last Raspbian Stretch adopts the new <a href='https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/' target='_blank'>Consistent Network Device Naming</a>. To correctly run Eclipse Kura on the Raspberry Pi, it should be disabled adding the ```net.ifnames=0``` parameter at the end of the /boot/cmdline.txt file." %}
 
-To install Kura with its dependencies on the Raspberry Pi, perform the
+To install Eclipse Kura with its dependencies on the Raspberry Pi, perform the
 following steps:
 
-1. Boot the Raspberry Pi with the latest Raspbian image (starting from release 2.1.0 Kura only supports Debian 8 or above).
-
-2. The dhcpcd5 package is not compatible with Kura and needs to be removed
-    performing the following command:
-
-    <pre>sudo apt-get purge dhcpcd5</pre>
-
-3. After removing dhcpcd5 package you need to enable network interface(s) and ensure internet access. Follow the instructions <a href="https://raspberrypi.stackexchange.com/questions/37920/how-do-i-set-up-networking-wifi-static-ip-address/74428#74428" target="_blank">here</a>.
-
-4. Install the gdebi command line tool:
-
-    <pre>sudo apt-get update
-   sudo apt-get install gdebi-core</pre>
-
-5. Make sure that Java 8 is installed with
-
-    <pre>java -version</pre>
+1. Boot the Raspberry Pi with the latest Raspbian image (starting from release 5.1.0 Kura only supports Debian 11 or above).
```

Comment:
Are we sure that previous raspbian builds do not work? I believe we can say that 5.1.0 is tested with bullseye, right?

Reply:
yeah, i'll modify.